share/completions/qjsc.fish

```diff
@@ -0,0 +1,26 @@
+# Define the completions for the qjsc command
+# helped by chatGPT
```

Comment:
Sorry, no.

I've seen chatgpt-generated code and it was absolutely wrong, but not in obvious ways.

That means I'm going to close this, and ask you to go over it manually.

I'm also going to ask you to run `fish_indent` on it, you'll be surprised.